### DISCUSSION TITLE
Wt32-sc01 plus how to start?

### DISCUSSION THREAD

**Discussion by sim1988son** [meshtastic/firmware#4686] [Q&A]
I don't know how to type lib into platformio to run wt32-sc01 plus. Need a code? Thanks a lot.

**Comment by Szetya** [comment]
I don't see much point in installing meshtastic on such a high resolution display.
You have a parallel addressing ST7796 display, currently TFTDisplay.cpp does not contain a class for it.
There are a limited number of GPIOs available (10-14, 21) which would possibly allow an SX1278 to be connected.
No I2C header so you will not be able to connect additional sensors.
Of course it is not impossible and could be used but I am sure it will not be finalized
